### DISCUSSION TITLE
What is the intended pattern for sidebar included files?

### DISCUSSION THREAD

**Discussion by ediphy-dcottrell** [docsifyjs/docsify#1476] [Q&A] [answered]
When you include a _sidebar.html

`* (here)[here.md]
`

And the in here.md you have

```
# A
# B
## C

```

The rendered result in the side bar ignores the first header. And if the first entry is a ## that it bumped to the top level. What is the intended usage pattern in these cases here? I have a feeling the first header is being stripped from the logic not just the display.

**Comment by sy-records** [comment, marked as answer]
In general, we align `here ` with the first level heading `A`

```
* (A)[here.md]

# A
# B
## C

```

![image](https://user-images.githubusercontent.com/33931153/104390781-73f87a80-5579-11eb-8c8f-0db80f5392bf.png)
